crac/src/test/groovy/io/micronaut/crac/HikariSpec.groovy

```diff
@@ -50,7 +50,7 @@ class HikariSpec extends Specification {
         hikariDataSource.running
 
         when: "we make sure we've got a connection warmed up"
-        def rows = new Sql(dataSource).rows("select 1")
+        def rows = new Sql(hikariDataSource).rows("select 1")
```

Comment:
Without this change we got:

```
  io.micronaut.data.connection.exceptions.NoConnectionException: No current connection present. Consider declaring @Connectable or @Transactional on the surrounding method
```

I *think* it's right...  @dstepanov does this look right?  Not sure why it started failing with M9